.github/workflows/ci.yaml

```diff
@@ -35,11 +35,18 @@ jobs:
         yarn run prettier
         yarn run build-tests
         npx ts-mocha -r tsconfig-paths/register ./@here/*/test/*.ts
+        yarn run build
+        yarn run typedoc
+    - name: browser test
+      run: |
+        set -ex
+        yarn
         firefox --version
         whereis firefox
         yarn test-browser --headless-firefox
         google-chrome --version
         whereis google-chrome
         yarn test-browser --headless-chrome
-        yarn run build
-        yarn run typedoc
+        if: matrix.os != macOS-latest
```

Comment:
What does this if do? Will it not execute the whole run section?

Reply:
I am testing it. The github support team told me this if should filter the whole section, but it is not working as expected. I have requested more feedback. I can't filter the rules by OS for some reason.

Reply:
Update: the `if` syntax works as expected and it lets you filter by os, so I will try to merge the three CI files into one so it is easier to maintain and we reuse code.